packages/office-ui-fabric-react/src/components/TeachingBubble/TeachingBubbleContent.base.tsx

```diff
@@ -92,7 +92,7 @@ export class TeachingBubbleContentBase extends BaseComponent<ITeachingBubbleProp
 
       headerContent = (
         <div className={classNames.header}>
-          <HeaderWrapperAs className={classNames.headline} id={ariaLabelledBy}>
+          <HeaderWrapperAs role="heading" className={classNames.headline} id={ariaLabelledBy}>
```

Comment:
you can specify level as well

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/heading_role

Reply:
Works for me. I'm not sure why it's better though...